fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1488,6 +1488,9 @@ public class Config extends ConfigBase {
     @ConfField(mutable = true, masterOnly = true, expType = ExperimentalType.EXPERIMENTAL)
     public static boolean enable_resource_group = true;
 
+    @ConfField(mutable = true, expType = ExperimentalType.EXPERIMENTAL)
+    public static boolean enable_query_queue = true;
```

Comment:
add docs

Reply:
I think we only need one config `enable_resource_group`.

Reply:
This would be removed after this feature is stable